crates/uv/src/commands/python/install.rs

```diff
@@ -61,21 +61,33 @@ pub(crate) async fn install(
     let installed_installations: Vec<_> = installations.find_all()?.collect();
     let mut unfilled_requests = Vec::new();
     for (request, download_request) in requests.iter().zip(download_requests) {
-        writeln!(
-            printer.stderr(),
-            "Searching for Python versions matching: {}",
-            request.cyan()
-        )?;
+        if matches!(requests.as_slice(), [PythonRequest::Any]) {
+            writeln!(printer.stderr(), "Searching for Python installations")?;
+        } else {
+            writeln!(
+                printer.stderr(),
+                "Searching for Python versions matching: {}",
+                request.cyan()
+            )?;
+        }
         if let Some(installation) = installed_installations
             .iter()
             .find(|installation| download_request.satisfied_by_key(installation.key()))
         {
-            writeln!(
-                printer.stderr(),
-                "Found existing installation for {}: {}",
-                request.cyan(),
-                installation.key().green(),
-            )?;
+            if matches!(request, PythonRequest::Any) {
+                writeln!(
+                    printer.stderr(),
+                    "Found existing installation: {}",
```

Comment:
I think in `uninstall` I just made this `"Found: {}"` since we don't need to show the request.